client/app/pages/queries/query.html

```diff
@@ -153,10 +153,14 @@ <h3>
                       <span ng-non-bindable>{{&nbsp;}}</span>
                     </button>
 
-                    <button type="button" class="btn btn-default" ng-click="formatQuery()" uib-tooltip="Format Query">
+                    <button type="button" class="btn btn-default m-r-5" ng-click="formatQuery()" uib-tooltip="Format Query">
                       <span class="zmdi zmdi-format-indent-increase"></span>
                     </button>
 
+                    <button type="button" class="btn btn-default m-r-5" ng-click="toggleAutoComplete()" uib-tooltip="Toggle AutoComplete" ng-class="onOrOff">
```

Comment:
Bootstrap already has a class for this - `active` - it makes button "pressed" and usually is used to implement such toggles. Also, usually "pressed" state is used to indicate that some feature is enabled, ad default state - when disabled (it's opposite behavior to implemented here).

Reply:
`ng-class="{active: autoCompleteQuery}"` and remove `onOrOff` + related code

Reply:
Updated.